validator-cli/src/ArbToEth/watcherArbToGnosis.ts

```diff
@@ -243,232 +306,90 @@ const watch = async () => {
-                      )) as Block
-                    ).timestamp;
-
-                    claim.timestampVerification = timestampVerification;
-                    claim.blocknumberVerification =
-                      logVerficiationStarted[logVerficiationStarted.length - 1].blockNumber;
-
-                    const claimHashCalculated = (await retryOperation(
-                      () => veaOutbox.hashClaim(claim, { blockTag: "latest" }),
-                      1000,
-                      10
-                    )) as string;
-                    if (claimHashCalculated != claimHash) {
-                      claim.honest = 1;
-                      const claimHashCalculated = (await retryOperation(
-                        () => veaOutbox.hashClaim(claim, { blockTag: "latest" }),
-                        1000,
-                        10
-                      )) as string;
-                      if (claimHashCalculated != claimHash) {
-                        console.error(
-                          "Invalid claim hash calculated for epoch " +
-                            veaEpochOutboxCheck +
-                            " claim " +
-                            claimHashCalculated +
-                            " expected " +
-                            claimHash
-                        );
-                        continue;
-                      }
-                    }
-                  }
-                }
-              } else {
-                console.log("claim " + veaEpochOutboxCheck + " is already challenged");
-                if (logChallenges[0].blockNumber < blockFinalizedGnosis.number) {
-                  veaEpochOutboxCheckClaimsRangeArray.splice(index, 1);
-                  index--;
-                  // the challenge is finalized, no further action needed
-                  console.log("challenge is finalized");
-                  continue;
-                } else {
-                  console.log(
-                    "challenge is not finalized yet, waiting for finalization to remove epoch from watch list."
-                  );
-                }
-                continue;
-              }
-
-              if (challengeTxnHashes.has(index)) {
-                const txnReceipt = (await retryOperation(
-                  () => providerGnosis.getTransactionReceipt(challengeTxnHashes.get(index)),
-                  10,
-                  1000
-                )) as TransactionReceipt;
-                if (!txnReceipt) {
-                  console.log("challenge txn " + challengeTxnHashes.get(index) + " not mined yet");
-                  continue;
-                }
-                const blockNumber = txnReceipt.blockNumber;
-                const challengeBlock = (await retryOperation(
-                  () => providerGnosis.getBlock(blockNumber),
+            )) as ContractTransaction;
+            console.log("Epoch " + veaEpochOutboxCheck + " challenged with txn " + txnChallenge.hash);
+            continue;
```

Comment:
Before `continue` the `challengeProgress.challenge.status` should be set to "pending"?

Reply:
Looks fixed 👍